manifest_template.yml

```diff
@@ -20,6 +20,12 @@ APIGEE_ENVIRONMENTS:
   name: int
   ratelimit: 20ps
   quota: 1200
+- display_name: Production
+  name: prod
+  approval_type: manual
+  ratelimit: '15000pm'        # todo: confirm with Spine/EPS team what these should be
```

Comment:
For now I'd probably match what we're using for int. 20 tps should be more than enough to start with